frontend/src/Commons/Header/Header.style.tsx

```diff
@@ -0,0 +1,60 @@
+import styled from '@emotion/styled';
+import * as tokens from '@bcgov/design-tokens/js';
+import { screenSizes } from '../../Constants/ScreenSize';
+
+const StyledHeader = styled.header`
+  display: flex;
+  align-items: center;
+  max-width: 100%;
+  max-height: 166px;
+  border-top-left-radius: 2px;
+  border-top-right-radius: 2px;
+  flex-direction: column;
+
+  /* Tablet */
+  @media (min-width: ${screenSizes.tablet}){
+
+  }
+  /* Desktop */
+  @media (min-width: ${screenSizes.desktop}){
+    flex-direction: row;
+    background-color: ${tokens.themeBlue100};
+    color: ${tokens.typographyColorPrimaryInvert};
+    border-bottom: ${tokens.layoutBorderWidthMedium} solid ${tokens.themeGold100};
+  }
+`;
+
+const Logo = styled.img`
+  max-height: 112px;
+  max-width: 123px;
+  order: 2; /* Moves logo below text */
+
+  /* Desktop */
+  @media (min-width: ${screenSizes.desktop}){
+    max-height: 123px;
+    max-width: 319px;
+    order: 1; /* Moves Logo before text */
+  }
+`;
+
+const Title = styled.h1`
+  flex: 1;
+  font-family: ${tokens.typographyFontFamiliesBcSans};
```

Comment:
Isn't `App.tsx` covering the font already?

Reply:
Nope, I'm not using App.css

Reply:
What do you mean?  From the files I see that App.tsx has a App.css with the font there:
![image](https://github.com/bcgov/nr-sustainment-capstone-2024/assets/80939533/2fa0624e-7e7d-4347-bd49-cc5f04057b19)

Are those styles not being applied?

Reply:
Just saw what you meant:
![image](https://github.com/bcgov/nr-sustainment-capstone-2024/assets/80939533/5329bfd7-2699-425f-90a1-969085dc9a1e)